src/e2e-test/resources/testdata/GCS/ExpectedOutputs/BQToDBAllDataTypes.json

```diff
@@ -0,0 +1,10 @@
+{"transaction_info":false,"transaction_num":[48,49,48,49,49,48,49,49],"transaction_uid":"30","transaction_date":18655,"transaction_dt":"2019-07-07T11:24:00","transaction_time":77160000000,"latitude":61.823765812,"unique_key":20062132134,"business_ratio":0.090480909,"updated_on":1552193401000000,"parent":["73e12a1c50f3ab2d7e299c56005e0695893c7dc9"],"inputs":{"input_script_bytes":[66,80,47,47,65,67,48,67,69,103,99,61],"input_script_string":"TestDATA","input_sequence_number":7245867295},"business_bigratio":52000000000000000000000000000000000000.00000000000000000000000000000000000000,"committer":{"name":"Ulrike Wagner","email":"87e0ca2bb46e13ca89eccbdabec7a29409c99475@mail.nih.gov","time_sec":1140728770,"tz_offset":0,"date":{"seconds":1140728770,"nanos":null}},"trailer":[{"key":"SVN-Revision","value":"1782","email":null}],"difference":[{"old_mode":33261,"new_mode":33261,"old_path":"update_fengoffice.sh","new_path":"update_fengoffice.sh","old_sha1":"f10eaa8e8a2b90d8e94bce7e13c585b5f8749f86","new_sha1":"ba3304fed713f82b08af699ada953b9c08b74e07","old_repo":null,"new_repo":null}],"Userdata":{"age":38,"company":"TestDATA"}}
+{"transaction_info":true,"transaction_num":[48,48,48,49,49,48,49,49],"transaction_uid":"10","transaction_date":18648,"transaction_dt":"2019-07-09T10:24:00","transaction_time":80760000000,"latitude":51.823765812,"unique_key":20082132134,"business_ratio":0.090480909,"updated_on":1520657401000000,"parent":["73e13a1c50f3ab2d7e299c56005e0695893c7dc9"],"inputs":{"input_script_bytes":[66,80,47,47,65,67,48,67,69,103,99,61],"input_script_string":"TestDATA","input_sequence_number":6245867295},"business_bigratio":72000000000000000000000000000000000000.00000000000000000000000000000000000000,"committer":{"name":"Mike","email":"87e0ca2bb46e13ca76eccbdabec7a29409c99475@mail.nih.gov","time_sec":1140338770,"tz_offset":1,"date":{"seconds":1140738770,"nanos":null}},"trailer":[{"key":"SVN-Revision","value":"1682","email":null}],"difference":[{"old_mode":44261,"new_mode":44261,"old_path":"update_fengoffice.sh","new_path":"update_fengoffice.sh","old_sha1":"f10eaa8e8a2b90d8e94bce7e13c585b5f8749f86","new_sha1":"ba3304fed713f82b08af699ada953b9c08b74e07","old_repo":null,"new_repo":null}],"Userdata":{"age":39,"company":"TestDATA"}}
```

Comment:
Why did you take same type of data 10 times? It's just values are different, but columns are same. Please try to make the data reusable.

Reply:
done 
commit id- defc54df110255ef539dea8d206481cc145b2fec